### PR TITLE
[RHDM-662] - Update RHDM templates on OCP and OSO

### PR DESCRIPTION
**Pull request by spolti**

### CODE REVIEW

File: official/rhdm/templates/rhdm70-kieserver-https-s2i.json [hunk 1]
```diff
@@ -3,16 +3,16 @@
     "kind": "Template",
     "labels": {
         "template": "rhdm70-kieserver-https-s2i",
-        "xpaas": "1.4.8"
+        "xpaas": "1.4.0"
```
Comment (bparees) [inline official/rhdm/templates/rhdm70-kieserver-https-s2i.json]: is this value correct/relevant?

Comment (errantepiphany) [reply official/rhdm/templates/rhdm70-kieserver-https-s2i.json]: Honestly I don't even know what this variable is used for... :/

Comment (bparees) [reply official/rhdm/templates/rhdm70-kieserver-https-s2i.json]: it's going to apply that label to all the resources created by the template.

i'm going to merge this as is, but if that label isn't relevant, remove it from the upstream template at some point.

Comment (spolti) [reply official/rhdm/templates/rhdm70-kieserver-https-s2i.json]: @bparees Thanks for pointing this out, we will remove it for the next releases.

**Pull request merged**